src/plugins/intel_gpu/src/kernel_selector/tensor_type.cpp

```diff
@@ -534,11 +534,22 @@ void DataTensor::SwapXY() {
     // Swap XY axes.
     y.pitch = 1;
     x.pitch = y.v + y.pad.Total();
+    int x_idx = Channelndex(l, DataChannelName::X);
+    int y_idx = Channelndex(l, DataChannelName::Y);
+    int f_idx = Channelndex(l, DataChannelName::FEATURE);
+    int b_idx = Channelndex(l, DataChannelName::BATCH);
+
+    if (x_idx < 0 || y_idx < 0 || f_idx < 0 || b_idx < 0) {
+        throw std::runtime_error("Invalid layout channel index");
+    }
```

Comment:
Please use `OPENVINO_ASSERT` instead of `if` + `throw`

Reply:
Applied it and thanks.